src/user/system/CertbotManager.ts

```diff
@@ -410,3 +420,24 @@ class CertbotManager {
 }
 
 export default CertbotManager
+
+export class CertCommandGenerator {
+    constructor(private rules: CertbotCertCommandRule[], private defaultCommand: string[]) {
+    }
+
+    private getCertbotCertCommandTemplate(domainName: string): string[] {
+        for (const rule of this.rules) {
+            if (rule.domain === '*'
+                || domainName === rule.domain
+                || domainName.endsWith('.' + rule.domain)
+            ) {
+                return rule.command ?? this.defaultCommand
+            }
+        }
+        return this.defaultCommand
+    }
+    getCertbotCertCommand(domainName: string, variables: Record<string, string> = {}): string[] {
```

Comment:
The only place this command gets called we have:
```
                const cmd = self.certCommandGenerator.getCertbotCertCommand(domainName, {
                    domain: domainName,
                    webroot: WEBROOT_PATH_IN_CERTBOT + '/' + domainName
                })
```


Looks like the flexibility in the args is really not needed and causes more confusion. Let's just change the signature to be explicit. i.e.,
```
    getCertbotCertCommand(domainName: string, webroot:string)
```